io.github.qtox.qTox.json

```diff
@@ -22,49 +22,87 @@
         "--device=all"
     ],
     "build-options": {
-        "cflags": "-O3 -DSQLITE_HAS_CODEC",
-        "cxxflags": "-O3"
+        "cflags": "-O3 -fPIC",
+        "cxxflags": "-O3 -fPIC"
     },
     "cleanup": [
         "/include",
         "/lib/pkgconfig",
         "/share/man"
     ],
     "modules": [
+		"shared-modules/SDL/SDL-1.2.15.json",
+		"shared-modules/gtk2/gtk2.json",
```

Comment:
What uses gtk2?

Reply:
In v1.16.3 gtk2 was used by one of our status icon implementations. It has been removed on master since, so won't be a dependency in the next release.